packages/compass/src/app/components/update-toasts.tsx

```diff
@@ -6,18 +6,14 @@ import {
   palette,
   useDarkMode,
   cx,
+  Link,
 } from '@mongodb-js/compass-components';
 import { openToast } from '@mongodb-js/compass-components';
 
 const containerStyles = css({
   display: 'flex',
   flexDirection: 'row',
-  div: {
-    display: 'flex',
-    flexDirection: 'column',
-    margin: 'auto',
-    padding: spacing[1],
-  },
+  gap: spacing[100],
```

Comment:
spacing[100]?

Reply:
the [new units](https://www.mongodb.design/component/tokens/example/), existing 1 (4px) is new 100

Reply:
TIL!